dpki/src/seed.rs

```diff
@@ -38,11 +40,33 @@ pub enum SeedType {
     Mock,
 }
 
+pub struct SeedContext {
+    inner: String,
```

Comment:
If this is non-public anyways, it might be more efficient to store this as the bytes... and use `b""` strings, a'la:

```rust
let inner: [u8; 8] = *b"HCDEVICE";
```

Reply:
done